cpp/src/aztec/honk/proof_system/prover.cpp

```diff
@@ -299,6 +300,17 @@ template <typename settings> void Prover<settings>::execute_relation_check_round
                                         sumcheck::GrandProductInitializationRelation>;
 
     // Compute alpha challenge
+    transcript.apply_fiat_shamir("zeta");
+
+    // TODO(Cody): This is just temporary of course. Very inefficient, e.g., no commitment needed.
+    Fr zeta_challenge = transcript.get_challenge_field_element("zeta");
+    barretenberg::polynomial pow_zeta = power_polynomial::generate_vector(zeta_challenge, key->n);
+    key->polynomial_cache.put("pow_zeta", std::move(pow_zeta));
+    std::span<Fr> pow_zeta_fetched = key->polynomial_cache.get("pow_zeta");
```

Comment:
Don't need to 'get' here do we? Can't we just call commit on `pow_zeta`

Reply:
You're right, this was just a temporary thing due to some issue (I think I was just trying to access after the `std::move`). Fixed now.